index.html

```diff
@@ -186,16 +189,17 @@ <h3>bthome-weather-station</h3>
 
         <h3 id="ble-advertising">BLE advertising in the BTHome format</h3>
         <p>
-          We assume that you understand the basics of BLE advertisements. If not, a good place
+          The BTHome format can best be explained with an example. We assume that you understand the basics of
+          BLE advertisements. If not, a good place
```

Comment:
Let's make this shorter and within parentheses.

> We assume that you understand the basics of BLE advertisements (if not, [read this](#))

Reply:
Done